pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "zeton-django"
+version = "0.1.0"
+description = "Backend application for @ProjektZeton"
+authors = ["Zetonteam <l.miotk.py@gmail.com>"]
+license = "Apache"
+readme = "README.md"
+package-mode = false
+
+[tool.poetry.dependencies]
+python = "^3.12"
+Django = "^4.2.13"
+psycopg = "^3.1.19"
+djangorestframework = "^3.15.1"
+django-cors-headers = "^4.3.1"
+pyjwt = "^1.7"
+six = "^1.16.0"
+pyyaml = "^6.0.1"
+uritemplate = "^4.1.1"
+drf-spectacular = "^0.27.2"
+djangorestframework-jwt-5 = "^1.13.0"
+pytest = "^8.2.0"
```

Comment:
In the ideal world, I think that test-related packages should not be included in the main dependencies. On the other hand, if we put them in the dev dependencies that complicates the build process a little bit, so let's keep it this way for now

Reply:
My thoughts exactly - in fact I did have `pytest` & `pytest-django` in `dev.dependencies` section but that failed the build. The next step in this automation process would be to use multi-stage build or other trick to prevent that from happening and actually have dev deps separated.